README.md
/*! ROOT/README.md */


Jesse Gangi's Online Resume
======================


## HELLO WORLD:


This is my Online Resume.  
I am continuously developing this live.  
Check back easily by bookmarking this in your favorite browser!  
Use a shortcut: Ctrl+D, Cmd+D


***


## STAMP / HEADER:


### Title:
Jesse Gangi's Online Resume

### Description/Subject:
This is my Online Resume.

### Keywords:
Jesse Gangi, Jesse, Gangi, Aerial Applications, Aerial, Applications, Duluth, Minnesota, MN, Superior, Wisconsin, WI, Twin Ports, Twin, Ports, Northland, Upper Mid-west, Upper, Mid-west, Mid, West, HTML5 Boilerplate, HTML5, Boilerplate, Twitter Bootstrap, Twitter, Bootstrap, Font Awesome, Font, Awesome, JQuery, TimelineJS, HTML, CSS, CSS3, Sass, SCSS, JS, JavaScript, PHP

### Author/Creator:
Ganginator | Jesse Gangi | http://jessegangi.com/ | (218) 310-2447 | jgangi@aerialapps.com | @Ganginator

### Copyright:
Copyright 2012 to 2014 Jesse Gangi. All Rights Reserved.

### Link:
http://jessegangi.com

### Life:
8/11/12 to Current


***


## TIMES:


/* ADD - TESTING - REQUIRES RESEARCH */  
EMPTY


***


## NOTES:


The old repository became corrupted on 6/18/13.  
I cloned this from the master branch.  
I marked other repository as 'CORRUPTED', and saved it for the history.

I'm using it for my Online Resume, my business Aerial Applications, Templating, and Examples.

I'm still modifying for personal use, and will remove unnecessary stuff as I go.

I'm still considering minifying everything when finished.  
However, I am having issues with getting Grunt to work on my server.

Contact me if you are interested in hiring me, with any errors, suggestions, or help.


## SPEC SPECIFIC NOTES:


#### This is an mix of:

HTML5 Boilerplate: http://html5boilerplate.com/  
Bootstrap: http://getbootstrap.com/  
Font Awesome: http://fontawesome.io/  
TimelineJS: http://timeline.knightlab.com/  
Bootstrapx Clickover: https://github.com/lecar-red/bootstrapx-clickover/

And my own code, including many references.


HTML5 does not require you to wrap your attributes in quotes, and you do not have to close your elements. However, I prefer the XHTML structured semantic approach, even though it can seem redundant.

DON'T FORGET TO CHANGE the 'lang' attribute in  
`<html class="no-js" lang="en">`  
IF YOUR LANGUAGE IS NOT English!

Place favicon.ico and apple-touch-icon.png in the root directory  
http://mathiasbynens.be/notes/touch-icons

I do not indent the body, use it as a wrapper, or indent any wrappers.

It is no longer necessary to use the type attribute in stylesheets, it is implied in HTML5!

AS ALWAYS, THE ORDER OF THE STACK IS IMPORTANT!!!

EMPTY fields are for referencing what elements the code contains, whether or not I am applying a style.

I also add some unnecessary DIVs, ID's, and Styles for referencing.

So don't be too surprised. ;)

IN DATABASE SITES, THE #page CONTENT AREA'S width, THE #page GLOBALS, AND THE content ARE DESIGNATED BY A DYNAMIC ?page=/?pagename= QUERY REFERENCING style, layout, AND content TABLES.

IN DATABASELESS SITES, THE #page CONTENT IS FETCHED WITH $_GET BASED ON ?page=, AND THE RESPECTIVE FILES ARE LOCATED IN ~includes/content; WHILE THE LAYOUT IS DETERMINED BY THE RESPECTIVE FILES LOCATED IN ~includes.

includes/css.php contains all CSS links for reference.

All JavaScript is inserted at the bottom, in #feet, so the pages load faster, except Modernizr for HTML5/CSS3 feature detection, and HTML5 Shiv for enabling HTML5 in IE. Technically Modernizer 1.5+ includes HTML5 Shiv.  
Modernizr 2 (re)introduced modular builds. After testing create a customized build (using only the tests you need) for a production-ready, minified, and compressed version.

HTML5 Shiv, and Respond.js for IE8 support of HTML5 elements, and media queries.  
WARNING: Respond.js doesn't work if you view the page via file://

Apple Touch Icons have relative support for Android.  
Precomposed = No Gloss

It is no longer necessary to use the type attribute in scripts, it is implied in HTML5!

includes/js.php contains all JavaScript links for reference.

Also, I added an image preloader.


***


## INFORMATION


#### CSS Definitions:


###### ID = Singular
e.g. A person's Identification (ID) is unique to one person.

###### Class = Multiple
e.g. There are many people in a class.

###### Padding = Inside
Defines the space inside the element between the content, and the element's border.

###### Margin = Outside
Defines the space outside the element between the element's border, and other elements.


#### Quick CSS px to em Conversions:

100% = 1 em ~= 16px ~= 14pt  
40px / 16 = 2.5em  
16 * 0.0625em = 1px  
22px = 16 * em  
22px / 16 = em  
px / 16 = em  
em = px / 16


#### Word Equations:

[ ( NEW - OLD ) / OLD ] * 100 = DIFFERENCE%


#### Flurry Regions:
Aftica, Asia, Central America, Europe, Middle East, North America, Oceania, South America

#### Standard Continents:
Africa, Antarctica, Asia, Australia, Europe, North America, South America

#### Super-class Continents:
Afro-Eurasia, America, Eurasia, Oceania

#### Common Tag List:
Refactored, Designed, Redesigned, Developed, Created, Curated, Hosted, HTML5, CSS3, PHP, JavaScript, Font Awesome, Drupal


***


## QUOTES:


Have more fun!

"It's always the padding..." - Ganginator  
"Added scripts, for added awesomeness." - Ganginator  
"IMHO, if you support IE 6, you are the reason we cannot have nice things." - Ganginator  
"Tactically, it's better to be small than big." - Ganginator  
"... but what do I know." - Ganginator  
"Stay cool." - Ganginator

Hey, I just met you, and this is crazy, but here's my number, so call me, maybe?


***


## LINKS:


### More ideas for your 'head':
http://h5bp.com/d/head-Tips  
http://en.wikipedia.org/wiki/Musikalisches_W%C3%BCrfelspiel  
http://en.wikipedia.org/wiki/Euler's_identity


***


## TO DO:


VERIFY profile:  
`<head profile="http://w3.org/2005/10/profile">`

CONSIDER CHANGING TO em's OR rem's  
http://blog.cloudfour.com/the-ems-have-it-proportional-media-queries-ftw

VERIFY NECESSITY OF ALL SHIMS!

DO I HAVE TO Place Apple Touch Icons in the ROOT directory?  
DO I HAVE TO Place Favicons in the ROOT directory?  
SHOULD I have one favicon.ico in the ROOT directory?


***


&hearts;  
TL;DR  
|-o-|  
EOD  
/ END




changelog.md
<!-- ROOT/drop/changelog.md -->


=== English ===


===== Changelog / Release Notes =====


Project: Jesse Gangi's Online Resume
URL: //jessegangi.com/
Birth: 2012-08-11
Updated Date: 2014-07-18
Maintainer: Jesse Gangi
Maintainer email: jgangi@aerialapps.com
Contributor: EMPTY
Contributor email: EMPTY
Description: Jesse Gangi's Online Resume

Status/Stage/Version/Release/Build/State: Live Alpha v1.4-a0.0908 Stable
Release Requestor: Jesse Gangi
Release Date Submitted: 2014-07-18
Release Date Approved: 2014-07-18
Release Description: This is an update, removing many unnecessary files.
Release Comments: Cleaning house.


LIFE:


8/11/12 to Current


/* UPDATE THESE */
RELEASE / UPDATE LIFE:

8/11/12 to 7/18/14

Years = 2
Months = 23
Days = 707
Hours = 
Minutes = 
Seconds = 


Changes in this version include:


== Added ==


YYYY-MM-DD firstName lastName <handle@domain.tld>
* ?


== Removed ==


YYYY-MM-DD firstName lastName <handle@domain.tld>
* ?


== Changed ==


YYYY-MM-DD firstName lastName <handle@domain.tld>
* ?


== Upcomming ==


/*! NOT USED YET */


YYYY-MM-DD firstName lastName <handle@domain.tld>
* ?


==== Documentation ====


2014-05-01 Jesse Gangi <jgangi@aerialapps.com>
* ROOT/README.md


YYYY-MM-DD firstName lastName <handle@domain.tld>
* ?


=== Tasks ===


2014-05-13 Jesse Gangi <jgangi@aerialapps.com>
1> Remove Duplicates in ROOT/README.md

2014-07-17 Jesse Gangi <jgangi@aerialapps.com>
1> Edit ROOT/README.md Errors


YYYY-MM-DD firstName lastName <handle@domain.tld>
* ?


==== Tasks Completed ====


2014-06-26 Jesse Gangi <jgangi@aerialapps.com>
1> Remove Duplicates in ROOT/README.md


YYYY-MM-DD firstName lastName <handle@domain.tld>
* ?


==== Wish List ====


YYYY-MM-DD firstName lastName <handle@domain.tld>
* ?


==== Wish List Items Completed ====


YYYY-MM-DD firstName lastName <handle@domain.tld>
* ?


==== Issue Tickets ====


/*! NOT IMPLIMENTED YET */


YYYY-MM-DD firstName lastName <handle@domain.tld>
* tid: ? - TYPE - ATTRIBUTES | ATTRIBUTES - DESCRIPTION


=== NOTES ===


YYYY-MM-DD firstName lastName <handle@domain.tld>
* ?


=== / English ===




sitemap.php
<!DOCTYPE html><!-- PAGE --><!-- HTML5 Declaration / -->


<!--[if lt IE 7]> <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
<!--[if IE 7]> <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
<!--[if IE 8]> <html class="no-js lt-ie9"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en"><!-- HTML5 Element --> <!--<![endif]-->
<!-- DON'T FORGET TO CHANGE THE ABOVE lang="??" IF YOUR LANGUAGE IS NOT English !!! -->


<!-- HTML5 does not require you to wrap your attributes in quotes, and you do not have to close your elements. I, however, prefer the XHTML Structured Semantic approach, even though it can seem redundant. -->


<!-- 
	Jesse Gangi
	jessegangi.com
	218 310 2447
	jgangi @ aerialapps.com
	@Ganginator
-->


<head>


	<?php
		include("includes/head.php")
	?>

	<title>. - : Jesse Gangi's Online Resume | Sitemap | Duluth, Minnesota : - .</title>
	
	
</head>


<body><!-- CODE --><!-- I do not indent the body, and I do not use it as a wrapper. -->


<!-- Prompt IE 6 users to install Chrome Frame. Remove this if you support IE 6.
chromium.org/developers/how-tos/chrome-frame-getting-started -->
<!--[if lt IE 7]>
	<p class="chromeframe">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com">upgrade your browser</a> or <a href="http://www.google.com/chromeframe/?redirect=true">activate Google Chrome Frame</a> to improve your experience.</p>
<![endif]-->


<!-- I do not indent any wrappers. In this case: #wrapper, #outer, and #inner. Technically #wrapper is unecessary. CONDSIDER REMOVING #wrapper -->
<!-- DISABLED WRAPPERS -->
<!-- <div id="wrapper" class="100center"> -->


<div id="outer" class="100center" style="width: 100%; text align: center; margin: 0 auto;">


	<!-- <div id="viewport"> --><!-- NEED TO FIGURE OUT VIEWPORTS! -->
	
	
		<!-- <h3><mark>&nbsp;Viewport&nbsp;</mark></h3> --><!-- HTML5 Element -->
		
		
	<!-- </div> --><!-- / #viewport -->
	
	
<div id="inner" class="100center" style="width: 600px; text-align: left;">


	<header>
	
	
		<?php
			include("includes/header.php")
		?>
		
		
	</header>
	
	
	<!-- DISABLED CONTENT 
	< ?php
		include("includes/layout.php")
	?>
	-->


	<h1>HTML5 Site Map</h1>
	
	<h3><a href="http://jessegangi.com">http://jessegangi.com</a> Last Updated: 6/14/13</h3>
	
	
	<dl>
		<dt><strong>/<br />
		5 pages</strong><br /></dt>
			<dd>Home &raquo; <a href="http://jessegangi.com/" title="Home">http://jessegangi.com/</a></dd>
			<dd>About &raquo; <a href="http://jessegangi.com/?pagename=about" title="About">http://jessegangi.com/?pagename=about</a></dd>
			<dd>
			
				<dl>
					<dd>
					
						<dl>
							<dt><strong>?pagename=about/<br />
							4 pages</strong></dt>
								<dd>Experience &raquo; <a href="http://jessegangi.com/?pagename=about/experience" title="Experience">http://jessegangi.com/?pagename=about/experience</a></dd>
								<dd>Portfolio &raquo; <a href="http://jessegangi.com/?pagename=about/portfolio" title="Portfolio">http://jessegangi.com/?pagename=about/portfolio</a></dd>
								<dd>Activities &raquo; <a href="http://jessegangi.com/?pagename=about/activities" title="Activities">http://jessegangi.com/?pagename=about/activities</a></dd>
								<dd>Education &raquo; <a href="http://jessegangi.com/?pagename=about/education" title="Education">http://jessegangi.com/?pagename=about/education</a></dd>
						</dl>
						
					</dd>
				</dl>
				
			</dd>
			<dd>Tools &raquo; <a href="http://jessegangi.com/?pagename=tools" title="Tools">http://jessegangi.com/?pagename=tools</a></dd>
			<dd>
			
				<dl>
					<dd>
					
						<dl>
							<dt><strong>?pagename=tools/<br />
							3 pages</strong></dt>
								<dd>Software &raquo; <a href="http://jessegangi.com/?pagename=tools/software" title="Software">http://jessegangi.com/?pagename=tools/software</a></dd>
								<dd>Code &raquo; <a href="http://jessegangi.com/?pagename=tools/code" title="Code">http://jessegangi.com/?pagename=tools/code</a></dd>
								<dd>Frameworks &raquo; <a href="http://jessegangi.com/?pagename=tools/frameworks" title="Frameworks">http://jessegangi.com/?pagename=tools/frameworks</a></dd>
						</dl>
						
					</dd>
				</dl>
				
			</dd>
			<dd>Links &raquo; <a href="http://jessegangi.com/?pagename=links" title="Links">http://jessegangi.com/?pagename=links</a></dd>
			<dd>Contact &raquo; <a href="http://jessegangi.com/?pagename=contact" title="Contact">http://jessegangi.com/?pagename=contact</a></dd>
	</dl>
	
	
	<div id="footer">
	
	
		<?php
			include("includes/footer.php")
		?>
		
		
	</div><!-- / HTML5 Tag -->


<!-- DISABLED WRAPPERS -->
</div><!-- / #inner .100center -->


</div><!-- / #outer .100center -->


<!-- </div> --><!-- / #wrapper .100center -->


<div id="feet"><!-- MINE -->


	<?php
		include("includes/feet.php")
	?>
	
	
</div><!-- / MY #feet -->


</body><!-- / CODE -->


</html><!-- / PAGE --><!-- / HTML5 Element -->